tests/test_reference_samplers.py

```diff
@@ -17,16 +17,75 @@
 
 import dimod
 from dwave.system.testing import MockDWaveSampler
-
 import hybrid
 from hybrid.samplers import QPUSubproblemAutoEmbeddingSampler
 from hybrid.reference.kerberos import KerberosSampler
+from hybrid.reference.lattice_lnls import LatticeLNLSSampler
+from hybrid.reference.lattice_lnls import LatticeLNLS
+from hybrid.decomposers import make_origin_embeddings
 from hybrid.reference.pa import (
     EnergyWeightedResampler, ProgressBetaAlongSchedule,
     CalculateAnnealingBetaSchedule, PopulationAnnealing
 )
 
 
+class MockDWaveSamplerGeneralization(MockDWaveSampler):
+    """Extend the `dwave.system.testing.MockDWaveSampler` to Pegasus topology.
+    
+    Adding topology and shape keywords to MockDWaveSampler for this purpose.
+    This function is mirrored in test_decomposers.py
```

Comment:
This class should not be needed anymore, since the upstream `MockDWaveSampler` now supports not only Pegasus, but also Zephyr topology.

`qpu_scale` is called `topology_shape` in current `MockDWaveSampler`.

This work can be done in a follow-up PR.